### ISSUE TITLE
Missing dependencies in dependencies.py

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#5027]
It seems that we are missing two dependencies:
- **PIL**, required by the GUI
- **pyasn1**, required by IPv8

Both of them can easily be installed with pip3.

**Comment by Solomon1732** [comment]
If I were to PR this, would the lines to add to the file in questions be something like this?
```python
{'module': 'PIL', 'install_type': 'pip3', 'package': 'PIL', 'optional': False, 'scope': 'gui'},
{'module': 'pyasn1', 'install_type': 'pip3', 'package': 'pyasn1', 'optional': False, 'scope': 'ipv8'},
```

**Comment by ichorid** [comment]
@Solomon1732 , it should go into `release-7.4.0` branch.

**Comment by Solomon1732** [comment]
This can be closed since #5041 was merged.
